### PR TITLE
Closes #203 - Transfer contract agnostic call

### LINKED ISSUE #203: Transfer Contract - Execute `call` parameter should be agnostic

### PR DESCRIPTION
**Pull request by vlopes11**
Closes #203

### CODE REVIEW

File: contracts/transfer/src/tests.rs [hunk 1]
```diff
@@ -284,20 +288,22 @@ fn withdraw_from_transparent_to_contract() {
...
-            spend_proof: spend_proof_stct,
-        }),
-    };
+    let call = Call::send_to_contract_transparent::<MemStore>(
+        bob_address,
+        bob_value,
+        bob_psk.A().into(),
+        spend_proof_stct,
+    )
+    .unwrap();
```
Comment (CPerezz) [inline contracts/transfer/src/tests.rs:297]: Use expect. In case we can capture the `panic` appended &str it could be used as err message.

**Review by CPerezz — APPROVED** [review]
LGTM

Ading a CHANGELOG for this crate in particular would not be a bad idea. But just a suggestion

File: contracts/transfer/src/tests.rs (hunk 1 above)
Comment (vlopes11) [reply contracts/transfer/src/tests.rs:297]: Cool, I'll add these expect and CHANGELOG on the next PR related to the integrated tests. Ok?

Comment (CPerezz) [reply contracts/transfer/src/tests.rs:297]: Perfect!

**Pull request merged**